CMakeLists.txt

```diff
@@ -136,6 +138,11 @@ if(WITH_GPU)
     endif(NOT WITH_DSO)
 endif(WITH_GPU)
 
+if(WITH_MKLDNN)
+    message(STATUS "MKLDNN_LIBRARY: ${MKLDNN_LIBRARY}")
```

Comment:
This message can be moved to `cmake/external/mkldnn.cmake`, to make the main `CMakeLists.txt` as concise as possible.